app/internal/page/app.go

```diff
@@ -481,6 +524,29 @@ func makeHandle(mux *http.ServeMux, logger Logger, store sessions.Store, localiz
 	}
 }
 
+func routeToLpa(mux http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		parts := strings.SplitN(r.URL.Path, "/", 4)
+		if len(parts) != 4 {
+			http.NotFound(w, r)
+			return
+		}
+
+		id, path := parts[2], "/"+parts[3]
+
+		r2 := new(http.Request)
+		*r2 = *r
+		r2.URL = new(url.URL)
+		*r2.URL = *r.URL
+		r2.URL.Path = path
+		if len(r.URL.RawPath) > 5+len(id) {
```

Comment:
Is the significance of `5` here `/lpa/`?

Reply:
yes, I will constantify it